apps/api/src/model/exif.ts

```diff
@@ -9,7 +9,10 @@ export default (sequelize: Sequelize) => {
         primaryKey: true,
         autoIncrement: true,
       },
-      shotTime: DataTypes.DATE,
+      shotTime: {
```

Comment:
nit: Unnecessary change, If the value of `allowNull` is not defined, the default value is true.